### PR TITLE
taxonomy: new ingredients for the Contribalim project

### PR DESCRIPTION
**Pull request by stephanegigandet**

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -7136,6 +7136,28 @@ en:cream cheese
 hr:krem sir
 pl:serek śmietankowy
 
+<en:cream cheese
+en:fresh cream cheese
+
+<en:fresh cream cheese
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:7142]:
```suggestion
<en:soft white cheese
```

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -7136,6 +7136,28 @@ en:cream cheese
...
+en:Petit suisse, Petits suisses
+fr:Petit suisse, Petits suisses
+xx:Petit suisse
+wikidata:en:Q585246
+agribalyse_proxy_food_code:en:19666
+ciqual_proxy_food_code:en:19666
+ciqual_proxy_food_name:en:Petit-Suisse like fresh cream cheese, plain, around 9% fat
+ciqual_proxy_food_name:fr:Fromage frais type petit suisse, nature, 10% MG environ
+
+<en:fresh cream cheese
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:7152]:
```suggestion
<en:soft white cheese
```

**Review by benbenben2 — APPROVED** [review]
I wonder if 
<en:cream cheese
should not be replaced by
<en:soft white cheese (fromage frais in FR).

What do you think?

**Comment by stephanegigandet** [comment]
> I wonder if
> <en:cream cheese
> should not be replaced by
> <en:soft white cheese (fromage frais in FR).

Maybe, I don't know. We can replace it. I don't really know the difference between "soft white cheese / fromage frais" and "cream cheese / fromage à la crème".

**Pull request merged**